### ISSUE TITLE
Hide sponsored posts & hide people you may know not functioning

### ISSUE THREAD

**Issue by nimkip** [indywidualny/FaceSlim#167]
Using version 2.5.2, neither setting changes my news feed, both ads and people you may know section are present. Is there anything I can provide to help diagnose the issue?

**Comment by indywidualny** [comment]
Some divs have to be located and added to the rules which make it hidden. If anyone contribute I would be pleased. I have no time for it now.

**Comment by pseudomonas** [comment]
Is there any chance you could share procedures with the likes of FB Purity? They're identifying divs for the purposes of hiding things, though not on mobile; I don't know what overlap there is.

**Comment by indywidualny** [comment]
Mobile version is actually sooo different than the desktop one. And they are still changing things. It's difficult to identify all the unwanted stuff. So this feature is actually not reliable and it always will be.

**Comment by masscream** [comment]
Hello, it seems that the this feature is still not working and would be very valuable! I understand that it's hard to follow still new and new FB standards, frames and junk.... Anyway, I believe it would help a lot if the app had the functionality to add its own filter pattern, like in uBlock Origin or Adblock filters. I can't block the frame on the hostname level in my Android AdBlocker... Thanks

**Comment by stephenjudge** [comment]
From my experience, the Hide Sponsored Post may be working, I don't see any sponsored  posts. However the Hide People You May Know, is definitely not working. Is there any likely fix for the latter on the horizon?
